### ISSUE TITLE
License

### ISSUE THREAD

**Issue by danruehle** [samuel/go-gettext#3]
I am interested in using this library but it does not have a license specified, so I really don't feel comfortable using it per the information at https://help.github.com/articles/open-source-licensing/. Would you consider adding a license to this project?

**Comment by ptman** [comment]
Yes, please license. Or does anyone know of a better Golang gettext library that works without cgo?
